test/unit/src/core/Object3D.tests.js

```diff
@@ -500,9 +500,30 @@ export default QUnit.module( 'Core', () => {
 
 		} );
 
-		QUnit.todo( "updateMatrix", ( assert ) => {
-
-			assert.ok( false, "everything's gonna be alright" );
+		QUnit.test( "updateMatrix", ( assert ) => {
+
+			const a = new Object3D();
```

Comment:
Do you folks think I should use `var` instead of `const` for the consistency in this file?

Reply:
IMO, `let` and `const` usage in tests is okay.